services/brig/src/Brig/IO/Intra.hs

```diff
@@ -548,7 +548,7 @@ createConnectConv from to cname mess conn = do
         . zUser from
         . maybe id (header "Z-Connection" . fromConnId) conn
         . contentJson
-        . lbytes (encode $ Connect to (messageText <$> mess) cname Nothing)
+        . lbytes (encode $ Connect to Nothing cname Nothing)
```

Comment:
Can the `message` field be removed from the `Connect` event type? This perhaps belongs in a separate PR, though.

Reply:
👍🏿 I added a futurework for another PR.